src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -303,7 +303,7 @@ export function getFlatIncidentKeys() {
   return incidentKeys;
 }
 
-export function getPtsdChangeText(changeFields) {
+export function getPtsdChangeText(changeFields = {}) {
```

Comment:
Are there unit tests for this function? Might be worth adding one for this case.

Reply:
Yes, good idea.

Reply:
@jbalboni - I added the test you asked for, could you take a look at this when you get a chance?